tests/dm_syncer/run.sh

```diff
@@ -32,6 +32,7 @@ function run() {
     dmctl_start_task $WORK_DIR/dm-task.yaml
 
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+    check_sync_diff $WORK_DIR $cur/conf/diff_config_blalist.toml
```

Comment:
we could check "block" part like 

https://github.com/pingcap/dm/blob/9ae67af8afc17fad62350685f03604a252dd370d/tests/all_mode/run.sh#L142-L144